test/Constraints/array_literal.swift

```diff
@@ -155,6 +155,13 @@ func defaultToAny(i: Int, s: String) {
 
   let a6 = [B(), C()]
   let _: Int = a6 // expected-error{{value of type '[A]'}}
+  
+  let a7: some Collection = [1, "Swift"]
+  // expected-warning@-1{{heterogeneous collection literal could only be inferred to '[Any]'; add explicit type annotation if this is intentional}}
```

Comment:
We are emitting the fix-it, so we need to verify them as well :)

Reply:
Oh, you're right! Will fix that :)

Reply:
Sure, I will add a check for fixit emission